tests/deploy/test_ci_deployments.py

```
@@ -0,0 +1,499 @@
+        logger.warning("No runtime image provided, using image from YAML")
+
+    # Create log directory for this test
+    log_dir = Path(request.node.name.replace("::", "_"))
+    log_dir.mkdir(parents=True, exist_ok=True)
+
+    # Deploy and test
+    async with ManagedDeployment(
+        namespace=deploy_namespace,
+        log_dir=str(log_dir),
+        deployment_spec=deployment_spec,
+    ) as deployment:
```

Ruff (0.14.13)

Local variable `deployment` is assigned to but never used

Remove assignment to unused variable `deployment`

(F841)